### DISCUSSION TITLE
Bug in my JavaScript Program

### DISCUSSION THREAD

**Discussion by ghost** [community/community#56144] [Programming Help]
### Body

I was working on a website that teaches you python, so I decided to add challenges, but when you click on any button it quickly displays “Try Again!” And displays and undisplays the message (unless you hold onto the button. If you hold the button the message just stays on the screen)  if you click any answer.
https://github.com/pytutorials/pytutorials.github.io/blob/main/challenge.js
https://github.com/pytutorials/pytutorials.github.io/blob/main/challenges/quiz.html

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
Sorry the code js code is at: https://pytutorials.github.io/quiz.js

**Comment by lennis-dev** [comment]
Hello again @pytutorials 

I think the problem is that you use for all questions the same element name use for the first question something like question1 and for the second question2 and so on, that would be your example for the first question:
``` HTML
<form>
    <input type="radio" name="question1" value="1">8</input>
    <input type="radio" name="question1" value="2">4</input>
    <input type="radio" name="question1" value="3">2</input>
    <input type="radio" name="question1" value="4">6</input>  
</form>
```

In JavaScript you can then read the value of the radio button like this: 

```JavaScript
const question1Value = document.querySelector('input[name="question1"]:checked').value;
const question1RightAnswer = 4;
if(question1Value == question1RightAnswer){
    //true
}else{
    //false
}
```
I could also contribute to your project either as a fork or as a contributor if you are interested write it here or create an issue in your repository and assign me to it.

Best regards
Lennis

**Reply by ghost** [reply]
Sure you can do that, thanks for the help!

**Reply by lennis-dev** [reply]
Are you inviting me as a contributor or should I fork the project and start a pull request?

**Reply by ghost** [reply]
I invited you
